docs/backends.md

```diff
@@ -112,6 +112,18 @@ better job.
 The only required parameter is the `path` we want to list or stream content
 from.
 
+### WebSocket
+
+WebSocket backend is **read-only** and can be used to get live logs.
+
+If you use the OVH's LDP, you can retrieve a WebSocket to test your data stream :
+
+> Data stream > Monitor in real time > Action > Copy WebSocket address
```

Comment:
I would mark this as a citation block (remark) pointing to the official documentation:

```suggestion
> If you use OVH's Logs Data Platform (LDP), you can retrieve a WebSocket URI to test your data stream by following instructions from the [official documentation](https://docs.ovh.com/gb/en/logs-data-platform/ldp-tail/#retrieve-your-websocket-address).
```

Reply:
✔️ I cropped it to three lines : 

```md
> If you use OVH's Logs Data Platform (LDP), you can retrieve a WebSocket URI to test your
> data stream by following instructions from the
> [official documentation](https://docs.ovh.com/gb/en/logs-data-platform/ldp-tail/#retrieve-your-websocket-address).
```